### PR TITLE
Works section project

### PR DESCRIPTION
**Pull request by karam084**
- Create a class (flex-botu).
- Create a class (resume).
- Create .flex-botu style
- Create .resume style.
- Create a list of work
- Create form and snapshot image
- Create 4 buttons. 
- Styles all positions. 
- Create four projects and added styles for them.
- Modified required for button and add a section for the group.

### CODE REVIEW

**Review by mke2111 — CHANGES_REQUESTED** [review]
### Hey @karam084, 

**This is a great project you have going on, well-done lad**. 

   
**However, there are a few sections to improve before your project is approved.** 🔴 
<hr>

## Required Changes ♻️
_Check the inline comments under the review for sections to improve._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 1]
```diff
@@ -58,5 +58,40 @@ <h1 class="my_name">I’m Karam Glad to see you!</h1>
...
+             <div class="paragraph"><p>A daily selection of privately personalized reads; no accounts or sign-ups required.</div>        
+             <div class="butto">
+                 <button class="htm">html</button>
+                 <button class="css-st">css</button>
+                 <button class="java-scr">javaScript</button>
+             </div>
+             <div class="see-proj">
+                 <button class="see-pr">See Project</button>
+             </div> 
+         </div>
```
Comment (mke2111) [inline index.html]: - [x] For this milestone, you are required to add the complete sections of projects and about me. Currently, from the template you have chosen, there should be 4 projects listed (you are having one project listed) and the about section comes after the projects section (your about section is coming before the projects section).
Screenshot_2021-09-24 My Portfolio

- Fix the order of the sections just like they are listed in the figma design.
- Add all four projects to your portfolio.
- Add a complete section for the 'About me' section.
- For a single project, left align all the contents.
- The button "See project" does not match the given design given in the figma template.
- Add complete styles to the about me section specifically the categories (languages, skills, frameworks). Make this section math the given design.

File: index.html [hunk 2]
```diff
@@ -58,5 +58,40 @@ <h1 class="my_name">I’m Karam Glad to see you!</h1>
              <div class="ellipse8"></div>
              <div class="ellipse15"></div>
          </div>
+         <div class="flex-botu">
+            <div class="resume">Get my resume</div>
+         </div>
```
Comment (mke2111) [inline index.html]: - [x] This should be a `button` tag

File: index.html [hunk 3]
```diff
@@ -58,5 +58,40 @@ <h1 class="my_name">I’m Karam Glad to see you!</h1>
...
+             <li><a href="#" class="css-icon"><img src="source/css-icon.png" alt="css-icon"><span>CSS</span></a></li> 
+         </ul>        
+         <ul class="fr-work">
+             <li class="border-frem">Frameworks</li>
+             <li class="border-skill">Skills</li>
+         </ul>
+         <div class="work-sec">
+             <div class="snapshoot-img"></div>
+             <ul class="tonic">
+                 <li>Tonic</li>
```
Comment (mke2111) [inline index.html]: - [x] This being the project title, use an `hx` tag

Comment (Mihndim2020) [inline index.html]: - [x] As requested by the previous code reviewer, please make the project titles (e.g. Tonic) are surrounded by an `<hx>` tags.

File: index.html [hunk 4]
```diff
@@ -58,5 +58,40 @@ <h1 class="my_name">I’m Karam Glad to see you!</h1>
              <div class="ellipse8"></div>
              <div class="ellipse15"></div>
          </div>
+         <div class="flex-botu">
+            <div class="resume">Get my resume</div>
```
Comment (Mihndim2020) [inline index.html]: - [x] As requested by the previous code reviewer, this is a button, please kindly fix that.

File: index.html (hunk 1 above)
Comment (Mihndim2020) [inline index.html:170]: - [ ] As requested by the former code reviewer, please implement the changes above. 

***See Design***
satD
- [x] For each project, align the content to the left as in the design. 
- [x] Add complete styles to the about me section specifically the categories (languages, skills, frameworks). Make this section match the given design.
- [OPTIONAL ] Please kindly style the buttons for enabled, hover and pressed states.
- [OPTIONAL] Please make sure your social media icons have uniform size.

**Review by Mihndim2020 — CHANGES_REQUESTED** [review]
## Hi @karam084 , 

Great work on making the changes requested by a previous reviewer 👏 👏 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.
stay-strong-youre-almost-there


 # Status: Requires Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by carlos-ssh — APPROVED** [review]
Hi @karam084 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**